.bazelrc

```diff
@@ -15,6 +15,9 @@
 # NOTE: This file is not loaded if google-cloud-cpp is built as part of a
 # larger project.
 
+build:linux --cxxopt=-std=c++14
+build:macos --cxxopt=-std=c++14
+
 # To workaround a bug in Bazel [1], gRPC requires this flag on macOS builds,
 # and there is (AFAIK) no way to "inherit" their definitions.
 #   [1]: https://github.com/bazelbuild/bazel/issues/4341
```

Comment:
Nit: Unnecessary difference in ordering between these two blocks here and in the generator/quickstart .bazelrc files?

Reply:
Fixed